doc/codi.txt

```diff
@@ -473,6 +473,12 @@ bin: mathjs
 notes:
   - Evaluation may not work until a newline is entered
 
+haxe~
+bin: ihx 
+maintainer: @grepsuzette
+notes:
```

Comment:
>(It has some display glitches here and there sometimes when typing, but I have been using it for years.)

Do you mind adding a note about that here?

Reply:
While using codi with python I see the same glitches, so I think we can ignore this :)